src/axom/inlet/docs/sphinx/example1_expected_documentation.rst

```diff
@@ -18,23 +19,28 @@ thermal_solver
      - Default Value
      - Range/Valid Values
      - Required
-   * - order
-     - thermal solver order
-     - 
-     - 1 to 2147483647
-     - |check|
+     - Value

```

Comment:
This column doesn't get output at all unless you set the option correct? Also, I find this a very confusing workflow.  The user would have to output this then to see the value in a helpful way they would have to render this w/ sphinx.

Reply:
I've removed this logic in favor of a `ConduitWriter` or something similar for dumping provided values